collector/mssql.go

```diff
@@ -246,1021 +244,1026 @@ type MSSQLCollector struct {
 	SQLAttentionrate              *prometheus.Desc
 	SQLCompilationsPersec         *prometheus.Desc
 	SQLReCompilationsPersec       *prometheus.Desc
-	UnsafeAutoParamsPersec        *prometheus.Desc
+
+	UnsafeAutoParamsPersec *prometheus.Desc
+
+	sqlInstances sqlInstancesType
 }
 
 // NewMSSQLCollector ...
 func NewMSSQLCollector() (Collector, error) {
-	sqlInstances = getMSSQLInstances()
 
 	const subsystem = "mssql"
 	return &MSSQLCollector{
+
 		// Win32_PerfRawData_{instance}_SQLServerAvailabilityReplica
 		BytesReceivedfromReplicaPersec: prometheus.NewDesc(
```

Comment:
Drop the `Persec` from the `Desc` field names as well

Reply:
done